pyproject.toml

```diff
@@ -6,48 +10,20 @@ readme = "README.md"
 requires-python = ">=3.8"
 license = {file = "LICENSE"}
 authors = [
-    {name = "D. Sommers", email = "dsommers@viktor.ai" }
+    {name = "VIKTOR", email = "support@viktor.ai" }
 ]
 maintainers = [
-    {name = "D. Sommers", email = "dsommers@viktor.ai" }
+    {name = "VIKTOR", email = "support@viktor.ai" }
 ]
 dependencies = [
     "viktor",
     "openpyxl",
     "plotly"
 ]
 
-[project.optional-dependencies]
-dev = [
-    "black",
-    "isort",
-    "pylint",
-]
 [project.urls]
 "Homepage" = "https://github.com/viktor-platform/excel_graph_parser"
 "Bug Reports" = "https://github.com/viktor-platform/excel_graph_parser/issues"
 
 [tool.setuptools]
 packages = ["excel_graph_parser"]
-package-data = {"excel_graph_parser" = ["*.jinja"]}

```

Comment:
Is this one not needed anymore?

Reply:
I don't think so because package-data is used to ship additional project files, but I don't see a jinja file anywhere